### DISCUSSION TITLE
FEAT: Conversation branching

### DISCUSSION THREAD

**Discussion by FranciscoMoretti** [mckaywrigley/chatbot-ui#1784] [Ideas]
Hey, awesome project!

Does conversation branching align with the future of this repo?

Similar to how ChatGpt lets you edit prompts, regenerate responses, and navigate through the conversation tree.
![image](https://github.com/mckaywrigley/chatbot-ui/assets/16997807/1be95135-e998-4941-ae71-93a266347563)

**Comment by crystal-coding-time** [comment]
I would love this feature to help guide models to the right responses.
